docs/source/components/nodes/imu.rst

```diff
@@ -0,0 +1,135 @@
+IMU
+===
+
+IMU (`intertial measurement unit <https://en.wikipedia.org/wiki/Inertial_measurement_unit>`__) node can be used to receive data from the IMU chip on the device.
+Our DepthAI devices use `BNO085 <https://www.ceva-dsp.com/product/bno080-085/>`__ 9-axis sensor (`datasheet here <https://www.ceva-dsp.com/wp-content/uploads/2019/10/BNO080_085-Datasheet.pdf>`__)
+that supports sensor fusion on the (IMU) chip itself. The IMU chip is connected to the Myriad X (VPU) over SPI (we have integrated
+`this driver <https://github.com/hcrest/bno080-driver>`__ to the DepthAI).
+
+
+How to place it
+###############
+
+.. tabs::
+
+  .. code-tab:: py
+
+    pipeline = dai.Pipeline()
+    imu = pipeline.create(dai.node.IMU)
+
+  .. code-tab:: c++
+
+    dai::Pipeline pipeline;
+    auto imu = pipeline.create<dai::node::IMU>();
+
+
+Inputs and Outputs
+##################
+
+.. code-block::
+
+  ┌──────────────┐
+  │              │
+  │              │      out
+  │     IMU      ├─────────►
+  │              │
+  │              │
+  └──────────────┘
+
+**Message types**
+
+- :code:`out` - :ref:`IMUData`
+
+Maximum frequencies
+###################
+
+Maximum output frequencies are 500 Hz raw accelerometer, 1000 Hz raw gyroscope values individually, and 500 Hz combined (synced) output.
+You can obtain the combined (synced) 500 Hz output with :code:`imu.enableIMUSensor([dai.IMUSensor.RAW_ACCELEROMETER, dai.IMUSensor.RAW_GYROSCOPE], 500)`.
+
+Usage
+#####
+
+.. tabs::
+
+  .. code-tab:: py
+
+    pipeline = dai.Pipeline()
+    imu = pipeline.create(dai.node.IMU)
+
+    # enable ACCELEROMETER and MAGNETOMETER_CALIBRATED at 100 hz rate
+    imu.enableIMUSensor([dai.IMUSensor.ACCELEROMETER, dai.IMUSensor.MAGNETOMETER_CALIBRATED], 100)
```

Comment:
let's add RAW_ACCELEROMETER + RAW_GYROSCOPE as examples, since we already have an example for that, and it's the most common use case.
same for cpp